l3kernel/l3tl-analysis.dtx

```diff
@@ -990,15 +990,15 @@
     \tl_if_exist:NTF #1
       {
         \exp_args:No \@@_analysis:n {#1}
-        \msg_show:nnxxxx { LaTeX / kernel } { show-tl-analysis }
+        \msg_show:nnxxxx { LaTeX / tl } { show-tl-analysis }
           { \token_to_str:N #1 } { \@@_analysis_show: } { } { }
       }
       { \tl_show:N #1 }
   }
 \cs_new_protected:Npn \tl_analysis_show:n #1
   {
     \@@_analysis:n {#1}
-    \msg_show:nnxxxx { LaTeX / kernel } { show-tl-analysis }
+    \msg_show:nnxxxx { LaTeX / tl } { show-tl-analysis }
       { } { \@@_analysis_show: } { } { }
   }
 %    \end{macrocode}
```

Comment:
why LaTeX/tl and kernel below?

Reply:
Back with the `show` business, which is fixed